core/tests/puppeteer-acceptance-tests/spec/voiceover-admin-tests/add-voiceover-artist-to-an-exploration.spec.ts

```
@@ -0,0 +1,110 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Utility functions for voiceover admin page if voiceover admin
+ * can add voiceover artist to an exploration
+ */
+
+import {UserFactory} from '../../puppeteer-testing-utilities/user-factory';
+import {VoiceoverAdmin} from '../../user-utilities/voiceover-admin-utils';
+import testConstants from '../../puppeteer-testing-utilities/test-constants';
+import {ConsoleReporter} from '../../puppeteer-testing-utilities/console-reporter';
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+const ROLES = testConstants.Roles;
+const invalidIdErrorToastMessage =
+  'Sorry, we could not find the specified user.';
+
+ConsoleReporter.setConsoleErrorsToIgnore([
+  new RegExp(
```

Please add an explanation, in a comment, for why we are ignoring this error.

---

Done